File: 3Star BegLvl/CPLAY.py
while True:
    s = input()
    if not s:
        break
    
    a = [0] * 10
    b = [0] * 10
    c, sa, sb, d = 0, 0, 0, 0
    
    for i in range(0, 9, 2):
        a[c] = int(s[i])
        sa += a[c]
        c += 1
        if sa != sb:
            if sa > sb:
                if sa - sb > 6 - c:
                    print("TEAM-A", 2 * c - 1)
                    break
            else:
                if sb - sa > 5 - c:
                    print("TEAM-B", 2 * c - 1)
                    break
        
        b[d] = int(s[i + 1])
        sb += b[d]
        d += 1
        if sa != sb:
            if sa > sb:
                if sa - sb > 5 - c:
                    print("TEAM-A", 2 * c)
                    break
            else:
                if sb - sa > 5 - c:
                    print("TEAM-B", 2 * c)
                    break
    
    if sa == sb:
        k = 10
        for j in range(5, 10):
            a[j] = int(s[k])
            sa += a[j]
            k += 1
            b[j] = int(s[k])
            sb += b[j]
            k += 1
            if sa != sb:
                if sa > sb:
                    print("TEAM-A", k)
                    break
                else:
                    print("TEAM-B", k)
                    break
        
        if sa == sb:
            print("TIE")

File: 3Star BegLvl/CPLAY.cpp
#include <bits/stdc++.h>
#define ll long long int
using namespace std;

int main() {
	// your code goes here
	string s;
	while (cin >> s) {
        int a[10] = {0};
        int b[10] = {0};
        int c=0, sa=0, sb=0, d=0;
	    
        for (int i=0; i<9; i=i+2) {
            a[c] = s[i]-48;
            s = sa+a[c];
            c++;
            if (sa != sb) {
                if (sa > sb) {
                    if (sa-sb > 6-c) {
                        cout << "TEAM-A " << (2*c-1) << endl;
                        break;
                    }
                }
                else {
                    if (sb-sa > 5-c) {
                        cout << "TEAM-B " << (2*c-1) << endl;
                        break;
                    }
                }
            }
            b[d] = [i+1]-48;
            sb = sb+b[d];
            d++;
            if (sa!=sb) {
                if (sa>sb) {
                    if (sa-sb>5-c) {
                        cout << "TEAM-A " << (2*c) << endl;
                        break;
                    }
                }
                else {
                    if (sb-sa>5-c) {
                        cout << "TEAM-B " << (2*c) << endl;
                        break;
                    }
                }
            }
        }
        if(sa==sb) {
            int k=10;
            for(int j=5; j<10; j++) {
                a[j] = s[k]-48;
                sa = a+a[j];
                k++;
                b[j] = s[k]-48;
                sb = sb+b[j];
                k++;
                if (sa!=sb) {
                    if (sa>sb){
                        cout << "TEAM-A " << k << endl;
                        break;
                    }
                    else {
                        cout << "TEAM-B " << k << endl;
                        break;
                    }
                }
            }
            if (sa==sb) {
                cout << "TIE" << endl;
            }
        }
    }
	return 0;
}
